app/views/pets/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_tag "/shelters/#{@shelter_pet.id}/pets" do %>
+
+<%= label_tag :image, "Image URL" %><br>
+<%= text_field_tag :image%><br>
+
+<%= label_tag :name %><br>
+<%= text_field_tag :name %><br>
+
+<%= label_tag :description %><br>
+<%= text_field_tag :description %><br>
+
+<%= label_tag :age, "Approximate Age" %><br>
+<%= text_field_tag :age %><br>
+
+<%= label_tag :sex %><br>
+<%= text_field_tag :sex %><br>
+
+<%= hidden_field_tag :status, 'Adoptable' %>
```

Comment:
Note to self: this gives any pet the status of 'adoptable' when they are created through the pets#new form